app/models/ApplicationForAccountHome.scala

```diff
@@ -34,8 +35,12 @@ final case class ApplicationForAccountHome(
   statusTag: Tag,
   actions: Seq[ActionItem]
 ) {
-  def dateString(): String =
-    ApplicationForAccountHome.formatter.format(date)
+  def dateString(lang: Lang = Lang("en")): String =
+    if (lang == Lang("en")) {
+      ApplicationForAccountHome.formatter.format(date)
+    } else {
+      ApplicationForAccountHome.welshFormatter.format(date)
+    }
```

Comment:
I'd have the default (else) as English, but not a major issue.

Reply:
the default case is english:  def dateString(lang: Lang = Lang("en")):

Reply:
it is the dfeault case def dateString(lang: Lang = Lang("en")):